ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -19,6 +20,29 @@
  */
 public class MessageScrubber {
 
+  static HashMap<String, String> ignoredNamespaces = new HashMap<String, String>() {
+
+    {
+      put("com-turkcell-yaani", "1612933");
+      put("org-mozilla-fenix-beta", "1612934");
+      put("org-mozilla-vrbrowser-dev", "1614410");
+      put("org-mozilla-fenix-performancetest", "1614412");
+      put("org-mozilla-vrbrowser-wavevr", "1614411");
+    }
+  };
```

Comment:
I'd prefer these be guava ImmutableMap, partly for safety of making sure we don't mutate them but mostly for prettier construction without the need for an anonymous subclass:

```
  private static final Map<String, String> IGNORED_NAMESPACES = ImmutableMap.<String, String>builder()
      .put("com-turkcell-yaani", "1612933") //
      .put("org-mozilla-fenix-beta", "1612934") //
      .put("org-mozilla-vrbrowser-dev", "1614410") //
      .put("org-mozilla-fenix-performancetest", "1614412") //
      .put("org-mozilla-vrbrowser-wavevr", "1614411") //
      .build();
```